styles/styles.css

```diff
@@ -0,0 +1,166 @@
+.container {
+    display: grid;
+    grid-template-rows: .5fr fr fr .75fr fr 1.5fr fr .5fr;
+}
+
+.header {
+    grid-template-rows: 1 / 2;
+    background-color: aqua;
+    width: 100%;
+    height: 4em;
+}
+
+
+nav ul {
+    display: grid;
+    grid-template-columns: 70% 15% 15%;
+    list-style-type: none;
+    font-weight: bold;
+}
+
+
+
+.title {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    font-weight: bold;
+    border-bottom: lightblue 2px solid; 
+    padding-bottom: 40px;  
+}
+
+
+.title sub {
+    padding: 10px 20px 50px 20px;
+    font-size: 1em;
+    font-weight: bold;
+    position: static; 
+    line-height: normal;
+}
+
+
+.title a {
+    border-style: outset;
+    border-style: solid;
+    border-width: 2px;
+    border-radius: 7px;
+    text-decoration: none;
+    padding: 10px;
+    color: orange;
+}
+
+.title a:hover {
+    background-color: lightcoral;
+    color: green;
+} 
+
+
+.quote {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    font-family: 'Slabo 27px', serif;
+    border-bottom: lightblue 2px solid;
+    font-style: italic;
+    font-size: 1.4em;
+    background-color: lightgray;
+}
+
+blockquote {
+   margin: 200px;
+}
```

Comment:
You can style your "" using the quote tag within blockquote. Ex.- blockquote {
    quotes: "\201C""\201D""\2018""\2019";
}